Assets/HoloToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs

```diff
@@ -94,24 +95,37 @@ private void MaybeInitAsServer()
                 return;
             }
 
+            StartCoroutine(InitAsServer());
+        }
+
+        private IEnumerator InitAsServer()
+        {
             Debug.Log("Acting as host");
             // StopBroadcast will also 'StopListening'
             StopBroadcast();
 
+            // Work-around when building to the HoloLens with "Compile with .NET Native tool chain".
+            // Need a frame of delay after StopBroadcast() otherwise clients won't connect.
+            yield return null;
```

Comment:
Hmm yeah finding out from Unity is there is something like ServerReady event or something akin would be nice.
Will keep this PR open till you hear back.

I don't think the change is bad but just want to understand more :)